src/_playground/App.scss

```diff
@@ -571,3 +571,134 @@ a.toc-link {
         }
     }
 }
+
```

Comment:
This is the block of styles that will eventually need to migrate to Fiori Fundamentals so all libraries can take advantage of them.